python-avd/pyavd/_eos_designs/structured_config/underlay/router_isis.py

```diff
@@ -92,15 +90,15 @@ def _isis_net(self: AvdStructuredConfigUnderlay) -> str | None:
         else:
             system_id = self.ipv4_to_isis_system_id(self.shared_utils.router_id)
 
-        isis_area_id = get(self._hostvars, "isis_area_id", default="49.0001")
+        isis_area_id = self.inputs.isis_area_id
         return f"{isis_area_id}.{system_id}.00"
 
     @cached_property
     def _is_type(self: AvdStructuredConfigUnderlay) -> str:
-        default_is_type = get(self._hostvars, "isis_default_is_type", default="level-2")
-        is_type = str(get(self.shared_utils.switch_data_combined, "is_type", default=default_is_type)).lower()
+        default_is_type = self.inputs.isis_default_is_type
+        is_type = default(self.shared_utils.node_config.is_type, default_is_type)
         if is_type not in ["level-1", "level-2", "level-1-2"]:
-            is_type = "level-2"
+            is_type = default_is_type
```

Comment:
this changes the behavior as it used to default to `level-2` and now it default to whatever is `isis_default_is_type`. Frankly it may be a bug fix but I am just pointing the difference of behavior to verify (also it could be a breaking change for someone who enters "toto" as `is_type`... but what can we do :) )

Reply:
we have valid values on the `isis_default_is_type` field so we should be good.

Reply:
valid_values ok. But what about the fact that now we use `default_isis_type` instead of plain "level-2" ? or is this check not needed at all? (if we have valid_values on is_type as well maybe we don't need this at all)

Reply:
we have valid values on both. So this check should not be needed at all.

Reply:
fixed